pgtap/pickDeliver/pickDeliver/pickDeliver-rows-check.sql

```diff
@@ -0,0 +1,50 @@
+\i setup.sql
+
+SELECT plan(10);
+
+
+-- Check whether the same set of rows is always returned
+
+PREPARE expectedOutput AS
+SELECT * FROM pgr_pickDeliver(
+    $$ SELECT * FROM orders ORDER BY id $$,
+    $$ SELECT * FROM vehicles $$,
+    $$ SELECT * from pgr_dijkstraCostMatrix(
+        ' SELECT * FROM edge_table ORDER BY id ', ARRAY[3, 4, 5, 8, 9, 11])
+    $$
+);
+
+PREPARE descendingOrder AS
+SELECT * FROM pgr_pickDeliver(
+    $$ SELECT * FROM orders ORDER BY id $$,
+    $$ SELECT * FROM vehicles $$,
+    $$ SELECT * from pgr_dijkstraCostMatrix(
```

Comment:
this makes no sence,
Test that  `pgr_dijkstraCostMatrix` return the same values all the time regardless of ordering.